lib/shared/drive-constraints.js

```diff
@@ -405,6 +437,11 @@ exports.getDriveImageCompatibilityStatuses = (drive, image) => {
       type: exports.COMPATIBILITY_STATUS_TYPES.ERROR,
       message: exports.COMPATIBILITY_STATUS_MESSAGES.LOCKED
     })
+  } else if (exports.isDriveSizeLarge(drive)) {
+    statusList.push({
+      type: exports.COMPATIBILITY_STATUS_TYPES.WARNING,
+      message: exports.COMPATIBILITY_STATUS_MESSAGES.LARGE_DRIVE
+    })
   } else if (!_.isNil(drive) && !_.isNil(drive.size) && !exports.isDriveLargeEnough(drive, image)) {
```

Comment:
Hmmm, should `isDriveSizeLarge` (which is a warning) be checked for _after_ `isDriveLargeEnough` (which is an error) ?

Hmmm number 2, if we have a drive which is both 'large' and 'system', I wonder if we should display both warning labels?

Reply:
That's true, we should considering we do the same with unrecommended drive/image size.